homeassistant/components/jellyfin/__init__.py

```diff
@@ -58,9 +58,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
-    hass.data[DOMAIN].pop(entry.entry_id)
+    unloaded = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+    if unloaded:
+        data = hass.data[DOMAIN].pop(entry.entry_id)
```

Comment:
it would be better to use runtime_data if possible. That may be outside the scope of the PR, but it could also resolve/prevent other issues

https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry?_highlight=runtime_dat

Reply:
I agree with this comment. However, in this case it fixes a bug, which we can ship in a patch release.

For that, we should leave other refactoring out of it.

But a good/nice follow-up PR opportunity to consider @j-stienstra!

../Frenck

Reply:
Thanks for the suggestion. I will look into it!